### ISSUE TITLE
Not Published Solution Appearing in Community Solution

### ISSUE THREAD

**Issue by pertrai1** [exercism/exercism.io#6502]
Not sure if this would be expected:

<img width="467" alt="Screen Shot 2022-08-16 at 5 03 05 PM" src="https://user-images.githubusercontent.com/442374/184986673-fc6ef90c-4571-4cf9-997b-38af981ba77b.png">

When I clicked on the card it took me to 404 - https://exercism.org/tracks/go/exercises/chessboard/solutions/stbnd

**Comment by iHiD** [comment]
Yeah, this is just showing for you as a way of quickly getting back to your solution. We could probably make that clearer, so I'll leave this open :)

**Comment by pertrai1** [comment]
One note is that the above screenshot is not my solution. Was not sure if you were referring to this being my solution. https://exercism.org/tracks/go/exercises/chessboard/solutions

**Comment by iHiD** [comment]
Hmmm, I see. That's not great then. Thanks for reporting. Maybe we've dropped some ElasticSearch update here (cc @ErikSchierboom)
